timeseries/src/autogluon/timeseries/models/gluonts/torch/models.py

```diff
@@ -496,13 +496,13 @@ class TiDEModel(AbstractGluonTSModel):
         Note that this is different from the `target_scaler` that is applied to the *entire time series*.
     max_epochs : int, default = 100
         Number of epochs the model will be trained for
-    batch_size : int, default = 64
+    batch_size : int, default = 256
         Size of batches used during training
     predict_batch_size : int, default = 500
         Size of batches used during prediction.
     num_batches_per_epoch : int, default = 50
         Number of batches processed every epoch
-    lr : float, default = 1e-3,
+    lr : float, default = 1e-4,
```

Comment:
Interesting!

Reply:
Did you also try values between 1e-3 and 1e-4?

Reply:
not really. but the difference between 1e-3 and 1e-4 was significant.